projects/vdk-plugins/vdk-ingest-http/src/vdk/plugin/ingest_http/ingest_http_plugin.py

```diff
@@ -19,6 +19,28 @@ def vdk_configure(config_builder: ConfigurationBuilder) -> None:
     """
     Here we define what configuration settings are needed for the Ingest HTTP plugin with reasonable defaults
     """
+    config_builder.add(
+        key="INGEST_OVER_HTTP_CONNECT_TIMEOUT_SECONDS",
+        default_value=None,
+        description="How many seconds to wait for connecting to the server",
+    )
+    config_builder.add(
+        key="INGEST_OVER_HTTP_READ_TIMEOUT_SECONDS",
+        default_value=None,
+        description="How many seconds to wait for the server to read data",
+    )
+    config_builder.add(
```

Comment:
Should we be having a configuration variable that accepts two completely different data types (boolean -- string file path)?

Reply:
seems requests is designed that way

Reply:
Ok.